09-deploy-vcva.tf

```diff
@@ -38,13 +38,34 @@ resource "null_resource" "deploy_vcva" {
     destination = "/root/deploy_vcva.py"
   }
 
+  provisioner "file" {
+    source      = "templates/extend_datastore.sh"
+    destination = "/root/extend_datastore.sh"
+  }
+
   provisioner "remote-exec" {
     inline = [
       "python3 /root/deploy_vcva.py",
       "sleep 60",
+    ]
+  }
+}
+
+
+resource "null_resource" "vsan_claim" {
+  depends_on = [null_resource.deploy_vcva]
+  count      = var.esxi_host_count == 1 ? 0 : 1
+  connection {
+    type        = "ssh"
+    user        = "root"
+    private_key = file("~/.ssh/id_rsa")
```

Comment:
We should replace this with:
`private_key = chomp(tls_private_key.ssh_key_pair.private_key_pem)`
This will eliminate the need for that id_rsa

Reply:
we'll also need to make sure the key gets written to the client machine or part of the output. We can address this as part of a later pr.

Reply:
Sounds good.